pkg/apis/v1beta1/aksnodeclass.go

```diff
@@ -41,8 +42,12 @@ type AKSNodeClassSpec struct {
 	ImageID *string `json:"-"`
 	// ImageFamily is the image family that instances use.
 	// +kubebuilder:default=Ubuntu2204
```

Comment:
@matthchr, @tallaxes, can we possibly change this to default `Ubuntu` as the generic family?

Seems incorrect to lock ourselves into this current pattern of fixing our default to a version in the long term?

This should have no behavior difference besides what shows up in the CRD spec, unless the customer sets FIPS, since currently without FIPS enabled, the behavior will map the generic `Ubuntu` family to `2204` anyways.

- Could be a follow up separate from Linda project though.

Reply:
Assuming that there is no backend difference today, I agree we can probably make this change.

Let's do it in a different PR though?